### PR TITLE
Landing

### PR DESCRIPTION
**Pull request by rez0nat0r**
Landing. Created rep, branch, and started files for project ART.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Home from "../src/componentsBasic/home/home"
+import "./app.scss";
+
+const App = () => {
+  return (
+    <>
```
Comment (vickierose) [inline src/app.js:8]: in this case you don't need to wrap Home into empty tag

File: src/componentsBasic/home/home-sections/home-Info/home-info.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      display: flex;
+    }
+    &__content {
+      width: 705px;
+      height: 716px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: flex-start;
+      padding-right: 11%;
```
Comment (vickierose) [inline src/componentsBasic/home/home-sections/home-Info/home-info.scss]: I wouldn't recommend to write paddings in %. Could you please explain why is that so or change it to px value

File: src/componentsBasic/home/home-sections/home-Info/home-info.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import "./home-info.scss";
```
Comment (vickierose) [inline src/componentsBasic/home/home-sections/home-Info/home-info.js]: files with components should be named with PascalCase, e.g App.js SomeComponent.js, HomeInfo.js

Also good practice is when file name and component name are the same, e.g. App.js => `const App = ({someProp}) => (<p>{someProp}</p>)`,
SomeComponent.js => `const SomeComponent= ({someProp}) => (<p>{someProp}</p>)`,
HomeInfo.js => `const HomeInfo= ({someProp}) => (<p>{someProp}</p>)`,

File: src/componentsBasic/home/home-sections/home-Info/home-info.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
+
+.info {
+    margin-bottom: 47px;
+    &__inner {
+      display: flex;
+    }
+    &__content {
+      width: 705px;
```
Comment (vickierose) [inline src/componentsBasic/home/home-sections/home-Info/home-info.scss]: For such a big block I'd rcommend to place sizes in % so it had basic responsiveness by default

File: src/componentsBasic/home/home-sections/home-subscribe/home-subscribe.scss [hunk 5]
```diff
@@ -0,0 +1,37 @@
+
+.subscribe {
+    margin-bottom: 95px;
+    &__inner {
+      display: flex;
+    }
+    &__content {
+      padding-left: 11%;
```
Comment (vickierose) [inline src/componentsBasic/home/home-sections/home-subscribe/home-subscribe.scss]: again: why are you using % for padding?

File: src/componentsUniversal/button/button.scss [hunk 6]
```diff
@@ -0,0 +1,8 @@
+@import '../../stylesUniversal/mixins/mixins.scss';
+
+.btn {
+    @include BTN (#ffffff, 600, #59abb0, none, 10px, flex, row, center, center);
```
Comment (vickierose) [inline src/componentsUniversal/button/button.scss]: this mixin is useless in overall, you need to pass too many arguments and you have only 2 possible button styles. You can just have classNames for them

File: src/componentsBasic/home/home.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import Footer from "../../componentsUniversal/footer/footer";
+import Header from "../../componentsUniversal/header/header";
+import Info from "./home-sections/home-Info/home-info";
+import Subscribe from "./home-sections/home-subscribe/home-subscribe";
+
+const Home = () => {
+  return (
+    <>
+      <Header/>
```
Comment (vickierose) [inline src/componentsBasic/home/home.js]: header and footer will be present on other pages too so it's better to move them 1 level up

File: src/componentsUniversal/footer/footer.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+        <div className="footer__content">
+          <p className="footer__content-text">
+            © Simple Art,Inc.2021
+            <br />
+            All rights reserved
+          </p>
+          <nav className="menu__footer">
+            <ul className="menu__footer-list">
+              <li className="menu__footer-item">
+                <a href="/" className="menu__footer-link">
```
Comment (vickierose) [inline src/componentsUniversal/footer/footer.js]: in react for inner routes that do not lead to another website we use react-router: https://reactrouter.com/en/v6.3.0/getting-started/overview and it's `Link` or `NavLink` component

File: src/componentsUniversal/footer/footer.scss [hunk 9]
```diff
@@ -0,0 +1,25 @@
+.footer {
+    background-color: #59abb0;
```
Comment (vickierose) [inline src/componentsUniversal/footer/footer.scss]: I'd suggest to move colors to css or sass variables so they were in one place and could be updated easily

File: src/componentsUniversal/header/header.js [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+const Header = () => {
+    return (
+        <header className="header">
+        <div className="container">
+          <div className="header__top">
+            <img className="header__img" src={logo} alt="logo-img" />
+            <nav className="menu">
+              <ul className="menu__list">
+                <li className="menu__item menu__item-header">
+                  <a href="/" className="menu__link menu__link-header menu__link-first">Courses</a>
```
Comment (vickierose) [inline src/componentsUniversal/header/header.js]: same here: look at react-router and replace `a` tags with it's Link or NavLink.
hint: NavLink is better for this case

File: src/componentsUniversal/header/header.scss [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+    list-style-type: none;
+    align-items: baseline;
+  }
+  &__item {
+    display: flex;
+    margin-right: 24px;
+  }
+  &__link {
+    &-first {
+      width: 86px;
```
Comment (vickierose) [inline src/componentsUniversal/header/header.scss]: it's better to have 1 common class for links, without numbering and give them common width and align it's content whithin the component

File: src/stylesUniversal/mixins/mixins.scss [hunk 12]
```diff
@@ -0,0 +1,11 @@
+@mixin BTN($color, $fw, $bcg, $border, $br, $flex, $f-direction, $j-content, $a-items) {
```
Comment (vickierose) [inline src/stylesUniversal/mixins/mixins.scss:3]: there's no use of this specific one mixin, but having this file is a good idea

**Review by vickierose — CHANGES_REQUESTED** [review]

**Review by vickierose — APPROVED** [review]

**Review by Nerevand — APPROVED** [review]

**Pull request merged**